docs/content/config/advanced/full-text-search.md

```diff
@@ -103,6 +103,60 @@ While indexing is memory intensive, you can configure the plugin to limit the am
     ```
     docker-compose exec mailserver doveadm fts optimize -A
     ```
+    Or as well as with [Spamassassin example][docs-faq-sa-learn-cron] you can use the internal instance of `cron` within `docker-mailserver` to avoid possible errors if the mail-server is not running:
+
+??? example
+
+    Create a _system_ cron file:
+
+    ```sh
+    # in the docker-compose.yml root directory
+    mkdir -p ./docker-data/dms/cron # if you didn't have this folder before
+    touch ./docker-data/dms/cron/fts_xapian
+    chown root:root ./docker-data/dms/cron/fts_xapian
+    chmod 0644 ./docker-data/dms/cron/fts_xapian
+    ```
+
+    Edit the system cron file `nano ./docker-data/dms/cron/fts_xapian`, and set an appropriate configuration:
+
+    ```conf
+    # Adding `MAILTO=""` prevents sending messages
+    # to emailafter completing cron task
+    MAILTO=""
+    #
+    # m h dom mon dow user command
+    #
+    # Everyday 4:00AM, optimize index files
+    0  4 * * * root  doveadm fts optimize -A
+    ```
+
+    Then with `docker-compose.yml`:
+
+    ```yaml
+    services:
+      mailserver:
+        image: docker.io/mailserver/docker-mailserver:latest
+        volumes:
+          - ./docker-data/dms/cron/fts_xapian:/etc/cron.d/fts_xapian
+    ```
+
+    Or with [Docker Swarm](https://docs.docker.com/engine/swarm/configs/):
+
+    ```yaml
+    version: '3.8'
+
+    services:
+      mailserver:
+        image: docker.io/mailserver/docker-mailserver:latest
+        # ...
+        configs:
+          - source: my_xapian_crontab
+            target: /etc/cron.d/fts_xapian
+
+    configs:
+      my_xapian_crontab:
+        file: ./docker-data/dms/cron/fts_xapian
+    ```
```

Comment:
Do you use Docker Swarm? I'm not sure if this is something we support/document much.

I only found [this original contribution for swarm to the SA FAQ entry](https://github.com/docker-mailserver/docker-mailserver/commit/70887618e2f4b5c100c262020097137b4a9422ec) (Jan 2019) and the [revised version](https://github.com/docker-mailserver/docker-mailserver/commit/aa7774801f224615369036bf3f929315cd2d499e) (Mar 2019).

I think we can discard this and assume a Docker Swarm user knows how to adapt from the `docker-compose.yml` example.

```suggestion
```

Reply:
No, I've looked at the documentation and examples to make sure the code in the documentation works. I agree with you that those who work with Docker Swarm will be able to do this part themselves.